tmk_core/common/action_layer.c

```diff
@@ -140,7 +140,7 @@ void layer_state_set(uint32_t state)
 
 /** \brief Layer clear
  *
- * FIXME: Needs docs
+ * Turn off all layers.
```

Comment:
It's great to see this, but could these changes be moved to a separate PR?

Reply:
Definitely!

Reply:
https://github.com/qmk/qmk_firmware/pull/4858